src/renderer/library/components/catalog/NoPublicationInfo.tsx

```diff
@@ -35,6 +36,16 @@ class NoPublicationInfo extends React.Component<IProps, undefined> {
                     <p>{__("catalog.noPublicationHelpL1")}</p>
                     <p>{__("catalog.noPublicationHelpL2")}</p>
                     <p>{__("catalog.noPublicationHelpL3")}</p>
+                    <p>{__("catalog.noPublicationHelpL4")}</p>
+                    {
+                        acceptedExtensionArray.map(
+                            (ext) => <span>
+                                {
+                                    ext
+                                }
+                            </span>,
```

Comment:
trailing comma? maybe array.join or array.reduce instead of array.map?

Reply:
it depends on the design

Reply:
but surely a trailing comma in the UI cannot be a good thing, right?

Reply:
@llemeurfr what do you prefer?

Reply:
I prefer no trailing comma

Reply:
Let's fix this small issue, and let's get this merged?

Reply:
I don't know what do you want ?
there is no trailing comma.
this displays the list of extensions.

Reply:
![NoComma](https://user-images.githubusercontent.com/625201/79326210-6343a700-7f0a-11ea-9c1e-be5d17ddd975.png)

You are correct! There is not trailing comma (I wonder what happens to the comma after the closing `span` element?)

Anyway, let's merge then :)

Reply:
JSX weirdness. Resolved.